redis-stack/redis-stack
# Redis 7.2.0

- RedisInsight 2.2.0 (#129)
- release rules changes (#128)
- removing edge, from release docker names (#132)
- redistimeseries 1.6.13 (#138)
- Update RedisBloom 2.2.17 (#141)
- Redis 7.0.2 (#142)
- Updating Redisinsight version for 2.4.0 (#144)
- Updating README with download links (#143)
- Adding the packages bucket as another release destination (#145)
- Unifying entrypoint script (#146)
- Adding helm chart to release process (#149)
- RedisTimeSeries 1.6.16 and RedisGraph 2.8.15 (#150)
- RediSearch 2.4.9 (#151)
- separating the manifests
- Arm snap image (#155)
- x86_64 AppImage Support (#154)
- nightly edge builds (#163)
- Upgrading build system to python 3.10 (#162)
- Fetch redis if built, or build (#166)
- redis 7.0.4 (#165)
- Updating RedisTimeSeries to 1.6.17 (#171)
- New RedisInsight version, and nodejs upgrade (#167)
- RediSearch 2.8.16, RedisGraph 2.4.11 (#174)
- Update RedisBloom v2.2.18 (#170)
- RedisJSON 2.2.0 (#175)
- RedisGraph 2.8.17 (#177)
- nightly credential check (#183)
- release paths for drafter (#184)
- README: highlight release process for RedisInsight
- Adding badges to README (#176)
- Green, forked PRs (#186)
- Fixing docker entrypoints for variable passing (#192)
- Ensuring armx linux, does not use jemalloc (#193)
- Adding README samples for running, and removing the double prefix on REDIS_ARGS (#194)
- Component upgrades (#196)
- adding link to latest and helm (#195)
- Labels with cleaner prefixes (#197)
- Updating the label for the helm chart.
- Quoting the paths in the entrypoint (#201)
- Upgrading FPM to 1.14.2 (#206)
- Support for archlinux packages (#205)
- Unit test to ensure versions are as set (#204)
- Consistency for dockergen arguments (#159)
- Create LICENSE (#207)
- Gathering test results (#208)
- Ensuring sha256 generating for missing packages (#209)
- New versions of modules (#211)
- Redis 7.0.5 (#220)
- marking docker as latest based on a variable (#221)
- ISLATEST is in caps, for the latest push. (#223)
- Updating instructions for the release process (#229)
- RedisBloom 2.4 (#233)
- cherry-pick 6.2 archlinux validation into master (#228)
- Amazonlinux 2 tests (#235)
- Tests for downloadable package on amazonlinux2 (#237)
- Updating release instructions (#236)
- Updating dependencies (#239)
- AppImage tests (#238)
- Fixing sha256 generating on redisinsight (#243)
- Versions of modules for October release (#245)
- Ubuntu Jammy support (#234)
- Jammy packages on ARM (#247)
- Ubuntu Focal ARM packages (#242)
- RedisInsight 2.12.0 (#246)
- Fixing docker builds (#254)
- Fix sha256 hashgen on OSX intel (#259)
- adding support for x.y.z branches with forced names (#261)
- License update (#264)
- Updating the docker publish action name (#269)
- Latest modules for master (#266)
- RedisInsight 2.14 [master] (#274)
- Moving to GITHUB_EVENT_PATH (#262)
- migrating arm docker to focal (#265)
- Removing some warnings and updating GitHub artifact versions (#244)
- Nightly edge docker (#285)
- Redis 7.0.6 (#279)
- module versions latest (#290)
- RedisJson 2.4.3 [master] (#294)
- default codeowner on module versions (#297)
- Moving Appimage base to bionic (#293)
- Building a single source for the redis/packagedredis version arguments (#304)
- Building redis on an m1, via ssh (#301)
- M1 package fix (#306)
- Moving latest check, into config (#305)
- Redis 7.0.7 (#307)
- Nightly edge built - use edge module versions (#303)
- Dispatchable nightly edge build (#312)
- Fixing nightly edge builds (#313)
- GPG signing linux tarballs (#314)
- codesigning readme updates (#316)
- Fixing nightly, edge docker to pull edge binaries (#318)
- Multi-platform docker build fix (#320)
- Support for retry fetching from s3 (#327)
- module version bumps, and insight (#321)
- remove redis-tack-server from redis-stack docker publish (#334)
- remove redis-tack-server-server from redis-stack-server docker publish (#337)
- json version bump 2.4.5 (#341)
- allow latest check to fail, since it can (#344)
- Fixing arm on-target compilation (#347)
- fpm upgrade (#351)
- adding support for manual CI runs (#354)
- Latest module versions (#353)
- Support for dedicated amazonlinux2 binaries (#352)
- RedisTimeSeries 1.8.9 (#359)
- Adding security and contribution guidelines (#302)
- Redis 7.0.10 (#368)
- Removing CODEOWNERS - it was never used (#375)
- Redis 7.0.11 (#377)
- Latest redis module versions for master (#386)
- Stack 7.2.0-M1 (#388)
- 7.2-rc2 (#395)
- Fixing token flow (#396)
- Version 7.2.0-M01 (#398)
- RedisInsight 2.24.0 (#399)
- [7.2] cherry picks from master (#408)
- 7.2.0-RC1 (#421)
- Cleaning up the config file (#422)
- [7.2] Cherry-pick gears and bump version for rc-2 (#427)
- redis version (#429)
- [7.2] latest redis-stack versions (#442)
- [7.2] Redis 7.2.0
